pages/signin.js

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 import Layout from "../src/components/layout";
+import { Typography } from "@material-ui/core";
 
 const Home = () => (
   <Layout title="Sign In">
-    <h1>Sign In</h1>
+    <Typography align="center" variant="h5" color="primary">
```

Comment:
This should look like an `h5` but it should be the next level down from the existing header for accessibility, so I believe this should actually be an `h2`.

To achieve that, I believe you'll keep `variant="h5"` but add `component="h2"`. Worth double-checking that in the browser, though.

Reply:
Yep, you are right. Added `component="h2"`